app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt

```diff
@@ -152,7 +160,7 @@ open class DefaultToolbarMenu(
             contentDescription = context.getString(R.string.browser_menu_share),
             iconTintColorResource = primaryTextColor(),
             listener = {
-                onItemTapped.invoke(ToolbarMenu.Item.Share)
+                onItemTapped.invoke(Item.Share)
```

Comment:
nit: Let's avoid changes like these in bugfix PRs.

It makes it easier on the reviewer, and also when some in the future needs to debug and look at git history or perform reverts. It's harder to manipulate changes then.

In the future, can just leave it as `ToolbarMenu.Item.Share` and all the other places.

Reply:
Agree let's address this!